proto/mls/message_contents/association.proto

```diff
@@ -0,0 +1,54 @@
+// Associations and signatures
```

Comment:
I think we might be able to leave this file deleted

Reply:
Cool